code-rules.tex

```diff
@@ -150,7 +152,7 @@ \section*{Tip 1: Consider your threat model}
   or your entire field having funding cut.
   These events affect so many people at the same time
   that the rest of the community can't absorb them.
-  Regional and national governments plans for disasters like these
+  Regional and national governments prepare for disasters like these
```

Comment:
fixes verb mismatch; also reduces repetitiveness as "plans" shows up twice later in the sentence.